.prettierignore

```diff
@@ -1,5 +1,9 @@
 # We can't have it in build/.prettierignore due to https://github.com/prettier/prettier/issues/15428
 # Textlint uses the same format, so we re-use the file.
 build/postgres-documentdb/documentdb/**
+build/legacy-mongo-shell/**
 internal/dataapi/api/openapi.json
 website/static/codapi/snippet.js
+website/src/components/HomepageFeatures/index.js
+website/babel.config.js
+website/versioned_docs/**
```

Comment:
Why are these files ignored? I think each ignored file should have explanation comment the first one.

Reply:
Those files should not be ignored